scripts/Makefile-gaf-reprocess

```diff
@@ -1,9 +1,22 @@
 BASE_NAMES = $(notdir $(wildcard /opt/go-site/annotations/*))
 
-all: $(foreach name,$(BASE_NAMES),/opt/go-site/annotations_new/$(name))
+all: $(foreach name,$(BASE_NAMES),/opt/go-site/annotations_new/$(name)) /opt/go-site/gaferencer-products/all.gaferences.json.gz
 
-/opt/go-site/annotations_new/%.gaf: /opt/go-site/annotations/%.gaf
+/opt/go-site/annotations_new/%.gz: /opt/go-site/annotations_new/%
+	pigz /opt/go-site/annotations_new/$*
+
+/opt/go-site/annotations_new/%.gaf: /opt/go-site/annotations/%.gaf /opt/go-site/gaferencer-products/all.gaferences.json
 	ontobio-parse-assocs.py -f $< -F gaf -o $@ -I /opt/go-site/gaferencer-products/all.gaferences.json --report-md /tmp/report.md --report-json /tmp/report.json convert -to gaf
 
-/opt/go-site/annotations_new/%.gpad: /opt/go-site/annotations/%.gpad
+/opt/go-site/annotations_new/%.gpad: /opt/go-site/annotations/%.gpad /opt/go-site/gaferencer-products/all.gaferences.json
 	ontobio-parse-assocs.py -f $< -F gpad -o $@ -I /opt/go-site/gaferencer-products/all.gaferences.json --report-md /tmp/report.md --report-json /tmp/report.json convert -to gpad
+
+/opt/go-site/annotations/%:
+	unpigz /opt/go-site/annotations/$*.gz
+
+/opt/go-site/gaferencer-products/all.gaferences.json.gz: /opt/go-site/gaferencer-products/all.gaferences.json
+	pigz /opt/go-site/gaferencer-products/all.gaferences.json
```

Comment:
I think you may be able to roll all `pigz` and `unpigz` steps into only 2 recipes, on to unzip, and one to zip. 
If every time we zip/unzip the product goes into the same directory, we can make a generic recipe:

```
%.gz: %
    unpigz $<

%: %.gz
    pigz $<

I think that should work?
```

Reply:
Genius!